cozepy/conversations/message/feedback/__init__.py

```
@@ -0,0 +1,111 @@
+        message_id: str,
+        **kwargs,
+    ) -> DeleteConversationMessageFeedbackResp:
+        url = f"{self._base_url}/v1/conversations/{conversation_id}/messages/{message_id}/feedback"
+        headers: Optional[dict] = kwargs.get("headers")
+
+        return self._requester.request(
+            "delete", url, False, cast=DeleteConversationMessageFeedbackResp, headers=headers
+        )
+
+
+class AsyncMessagesFeedbackClient(object):
```

Pylint (3.3.7)

[refactor] Class 'AsyncMessagesFeedbackClient' inherits from object, can be safely removed from bases in python3

(R0205)